protocol/messenger_communities.go

```diff
@@ -2041,6 +2041,13 @@ func (m *Messenger) LeaveCommunity(communityID types.HexBytes) (*MessengerRespon
 		}
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	defer cancel()
+	_, err = m.BackupData(ctx)
```

Comment:
I don't think it's worth forcing a backup. The loop will take care of it.

Reply:
Did you mean by loop, every 8h? This might not happen upon leaving a community. 
Are you saying it's okay to not force the backup and wait for the worse case 8h to have a usual back up?

Reply:
> Did you mean by loop, every 8h?

Yeah, the `startBackupLoop`.

> Are you saying it's okay to not force the backup and wait for the worse case 8h to have a usual back up?

Yes. Nowhere else do we force a backup. I think it's ok to let the loop do it's job. I tried to check. I thought we had some places we we do a `syncX` when we do a change, but I can't find it.

So if we wanted to force a backup, we'd have to do it only for the community and not for all the data.

Reply:
I had a look at the forcing of community only backup, but it seems to be require some considerable refactoring effort for having community only backup which I believe should be out of the scope of this PR to avoid a very long lasting PR.
I will leave the `startBackupLoop` take care of the backup for this PR and will open a ticket for handling the detangling of backup of community from other fields (if we really need to do that)